sktime/performance_metrics/forecasting/probabilistic/_classes.py

```diff
@@ -85,8 +95,42 @@ def _evaluate_by_index(self, y_true, y_pred, multioutput, **kwargs):
             )
         return out_series
 
+    def _check_alpha(self, alpha):
+        if isinstance(alpha, list):
+            alpha = np.array(alpha)
+        if isinstance(alpha, np.ndarray):
+            self.multiple_alphas = True
+            if not all(((alpha > 0) & (alpha < 1))):
+                raise ValueError("Alpha must be between 0 and 1.")
+            return alpha
+        if isinstance(alpha, float):
+            self.multiple_alphas = False
+            if not ((alpha > 0) & (alpha < 1)):
+                raise ValueError("Alpha must be between 0 and 1.")
+            return [alpha]
+        raise TypeError("Alpha should be a float or numpy array")
 
-class PinballLoss(_BaseProbForecastingErrorMetric):
+    def _check_ys(self, y_true, y_pred, multioutput):
+        if multioutput is None:
+            multioutput = self.multioutput
+        _, y_true, y_pred, multioutput = _check_reg_targets(y_true, y_pred, multioutput)
+        valid, msg, metadata = check_is_scitype(
+            y_pred, scitype="Proba", return_metadata=True, var_name="y_pred"
+        )
+        if not valid:
+            raise TypeError(msg)
+        y_pred_mtype = metadata["mtype"]
+        inner_y_pred_mtype = self.get_tag("scitype:y_pred")
+        y_pred_inner = convert(
```

Comment:
This is extremely useful. One thing that popped up in testing was that if I am converting quantiles -> intervals and there is a 0.5 quantile, it produces a column for 0 coverage. This is redundant so it may be worth adding a case distinction to not include this column when quantile is 0.5.

Reply:
Also as far as I could work out, the conversions assume that coverage is using symmetric quantiles, i.e. 0.9 coverage gives quantiles for 0.05 and 0.95 but it could correspond to 0.01 and 0.91 quantiles?

Reply:
Is there ever a case where a half interval is useful? Sometimes the conversion from quantiles to intervals can only find a lower bound, is this useful on its own or should  incomplete intervals be removed?

Reply:
> Is there ever a case where a half interval is useful?

I think the only "frequent" use case is applying a quantile loss to predictive intervals, the other way round does not seem sensible (although I included it for completeness). Similarly, the only conversion that I see being used frequently is from intervals to quantiles, not the other way round.

Reply:
>Also as far as I could work out, the conversions assume that coverage is using symmetric quantiles, i.e. 0.9 coverage gives quantiles for 0.05 and 0.95 but it could correspond to 0.01 and 0.91 quantiles?

Do they? I thought I changed that in the PR, then you get "half intervals". Only on `main`, symmetric quantiles are still assumed.

Reply:
> This is redundant so it may be worth adding a case distinction to not include this column when quantile is 0.5.

Well, it´s a degenerate case, and anyway converting from intervals to quantiles is the more frequent case. In that, you usually do not have a "zero width" interval.